README.md

```diff
@@ -91,4 +92,4 @@ Raghav Sharma - Link sharing and awareness creation for the project.
 ### Note
 
 * The parsers support parsing of any IPU Result of any course, so raise an issue if you want your course result to be displayed here.
-* Works best on Chrome and Firefox.
\ No newline at end of file
+* Works best on Chrome and Firefox.
```

Comment:
Squash and merge dude

Reply:
npm test is  just for general information

Reply:
Squashing isn't the point here, you could as well have made a commit per line then. I want us to have a rebased history instead of merging based ones. This helps maintain a clearer picture of people's contributions.